src-tauri/src/auto_launcher.rs

```diff
@@ -146,6 +222,9 @@ impl AutoLauncher {
         let auto_launcher = AutoLauncher::build_auto_launcher(app_name, &app_path)?;
 
         AutoLauncher::toggle_auto_launcher(&auto_launcher, is_auto_launcher_enabled)?;
+        #[cfg(target_os = "windows")]
+        self.toggle_windows_admin_auto_launcher(is_auto_launcher_enabled)
+            .await?;
```

Comment:
Feels like this is maybe something that should be withing `AutoLauncher::toggle_auto_launcher` which already does OS specific things for auto launching?